pkg/diagnostics/report.go

```diff
@@ -0,0 +1,33 @@
+package diagnostics
+
+import (
+	"github.com/noobaa/noobaa-operator/v5/pkg/bundle"
+	"github.com/noobaa/noobaa-operator/v5/pkg/options"
+	"github.com/noobaa/noobaa-operator/v5/pkg/util"
+	"github.com/spf13/cobra"
+	appsv1 "k8s.io/api/apps/v1"
+)
+
+// RunReport runs a CLI command
+func RunReport(cmd *cobra.Command, args []string) {
```

Comment:
The function should run a bunch of functions that will diagnose each topic
We should get the proxy check part into a different function and call it here.

Reply:
Done, Thanks